crates/astria-sequencer/src/utils.rs

```diff
@@ -16,9 +20,29 @@ impl<'a> std::fmt::Display for Hex<'a> {
     }
 }
 
+pub(crate) fn anyhow_to_eyre(anyhow_error: anyhow::Error) -> astria_eyre::eyre::Report {
+    let anyhow_result = Err::<(), _>(anyhow_error);
+    let boxed: Result<(), Box<dyn std::error::Error + Send + Sync>> =
+        anyhow_result.map_err(std::convert::Into::into);
+    let Err(err) = boxed else {
+        panic!("anyhow_to_eyre called on `Ok`")
+    };
+    eyre!(err)
+}
+
+pub(crate) fn eyre_to_anyhow(eyre_error: astria_eyre::eyre::Report) -> anyhow::Error {
+    let eyre_result = Err::<(), _>(eyre_error);
+    let boxed: Result<(), Box<dyn std::error::Error + Send + Sync>> =
+        eyre_result.map_err(std::convert::Into::into);
+    let Err(err) = boxed else {
+        panic!("eyre_to_anyhow called on `Ok`")
+    };
+    anyhow::anyhow!(err)
```

Comment:
Similar to the eyre case above: the err -> result -> err is probably not necessary. :)